pkg/kubefed2/unjoin.go

```diff
@@ -183,12 +193,15 @@ func UnjoinCluster(hostConfig, clusterConfig *rest.Config, federationNamespace,
 		removeFromClusterRegistry(hostConfig, clusterNamespace, clusterConfig.Host, unjoiningClusterName, dryRun)
```

Comment:
is this `clusterConfig.Host` safe?

Reply:
this `host` argument seems unused. i guess you can just remove it.

Reply:
Good catch. Thanks for your comments. It is resolved.